src/components/components/ReinforcementStatement/ReinforcementStatement.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./ReinforcementStatement.scss";
+import ReinforcementImage from "./Assets/reinforcementImage.jpg";
+
+function ReinforcementStatement() {
+  return (
+  <div className="Reinforcement">
+            <div className="Reinforcement__image">
+                <img src='./reinforcementImage.jpg' alt=' Reinforcement Image'/>
```

Comment:
Please import your assets like this: https://stackoverflow.com/a/45387838

Reply:
Changed image import style

Reply:
change  alt=' Reinforcement Image' to reinforcementImage may be? vs code says "Screen-readers already announce `img` tags as an image. You don’t need to use the words `image` (or any specified custom words) in the alt prop"